recipe/meta.yaml

```diff
@@ -26,43 +25,41 @@ requirements:
     - meson
     - ninja
   host:
-    - libdrm
-    # Uncomment when merged: https://github.com/conda-forge/staged-recipes/pull/25919
-    # - libglvnd
-    - wayland
-    - libxcb
-    - xorg-libx11
-    - xorg-libxext
-    - xorg-libxfixes
+    - libdrm           # [linux]
+    - libgl-devel      # [linux]
+    - libegl-devel     # [linux]
+    - wayland          # [linux]
+    - libxcb           # [linux]
+    - xorg-libx11      # [linux]
+    - xorg-libxext     # [linux]
+    - xorg-libxfixes   # [linux]
   run:
-    - wayland-protocols
-    - xorg-libx11
-    - xorg-libxext
-    - xorg-libxfixes
+    - wayland-protocols  # [linux]
+    - xorg-libxfixes     # [linux]
 
 test:
   commands:
-    - test -f $PREFIX/include/va/va.h
-    - test -f $PREFIX/lib/libva.so
-    - test -f $PREFIX/lib/libva.so.2
-    - test -f $PREFIX/lib/pkgconfig/libva.pc
-    - test -f $PREFIX/include/va/va_drm.h
-    - test -f $PREFIX/lib/libva-drm.so
-    - test -f $PREFIX/lib/libva-drm.so.2
-    - test -f $PREFIX/lib/pkgconfig/libva-drm.pc
+    - test -f $PREFIX/include/va/va.h                    # [unix]
+    - test -f $PREFIX/lib/libva.so                       # [unix]
+    - test -f $PREFIX/lib/libva.so.2                     # [unix]
+    - test -f $PREFIX/lib/pkgconfig/libva.pc             # [unix]
+    - test -f $PREFIX/include/va/va_drm.h                # [unix]
+    - test -f $PREFIX/lib/libva-drm.so                   # [unix]
+    - test -f $PREFIX/lib/libva-drm.so.2                 # [unix]
+    - test -f $PREFIX/lib/pkgconfig/libva-drm.pc         # [unix]
     # Uncomment when merged: https://github.com/conda-forge/staged-recipes/pull/25919
     # - test -f $PREFIX/include/va/va_glx.h
     # - test -f $PREFIX/lib/libva-glx.so
     # - test -f $PREFIX/lib/libva-glx.so.2
     # - test -f $PREFIX/lib/pkgconfig/libva-glx.pc
```

Comment:
This portion is ready to be uncommented. Include `libglx-devel` in the dependencies as well.